core/Enclave/enclave_hsm.cpp

```diff
@@ -281,7 +302,19 @@ sgx_status_t enclave_create_rsa_key(uint8_t *cmk_blob,
     sgx_status_t ret = SGX_ERROR_UNEXPECTED;
 
     // get keybloblen
-    uint32_t real_keyblob_size = sgx_calc_gcm_data_size(0, sizeof(rsa_key_data_t));
+    int32_t key_size;
+    switch (keyspec)
+    {
+        case EH_RSA_2048:
+            key_size = 2105;
```

Comment:
do not use hardcode!
it's a MUST requirement for the following patches...

Reply:
and also need to describe the details how you get/set this values..

Reply:
yes
I'll add some soon